tests/reg_tests/reg_default_options.py

```diff
@@ -30,12 +30,12 @@
     # Physics Paramters
     'discretization':'central plus scalar dissipation',
     'coarsediscretization':'central plus scalar dissipation',
-    'limiter':'vanalbeda',
-    'smoother':'runge kutta',
-    'equationtype':'euler',
+    'limiter':'van Albada',
```

Comment:
These options dictionary should be instantiated using `caseInsensitiveSet` to avoid duplicate entries (this was a bug I ran into a while ago which motivated this change)

Reply:
I tried this and it breaks things elsewhere in the tests. I think we should leave this for a later PR.

Reply:
I'll open an issue for this.